Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANSILLParameterScan.py

```diff
@@ -0,0 +1,242 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+from __future__ import (absolute_import, division, print_function)
+
+from mantid.api import *
+from mantid.kernel import *
+from mantid.simpleapi import *
+from SANSILLAutoProcess import needs_loading, needs_processing
+
+
+class SANSILLParameterScan(DataProcessorAlgorithm):
```

Comment:
Since this loads many files, I suggest to change to `PythonAlgorithm`, so that it doesn't carry the huge unrolled history of each of the files loaded.